app/packages/core/src/components/ColorModal/ColorModal.tsx

```diff
@@ -41,27 +68,75 @@ const ColorModal = () => {
           aria-labelledby="draggable-color-modal"
         >
           <Draggable bounds="parent" handle=".draggable-colorModal-handle">
-            <Container>
-              <DraggableModalTitle className="draggable-colorModal-handle">
-                <div style={{ margin: "4px" }}>Edit color scheme</div>
-                <CloseIcon
-                  onClick={() => setActiveColorModalField(null)}
-                  onMouseDown={(e) => e.stopPropagation()}
-                  style={{ margin: "4px", cursor: "pointer" }}
-                />
-              </DraggableModalTitle>
-              <DraggableContent>
-                <SidebarList />
-                <Display>
-                  {field === ACTIVE_FIELD.global && <GlobalSetting />}
-                  {field === ACTIVE_FIELD.json && <JSONViewer />}
-                  {typeof field !== "string" && field && (
-                    <FieldSetting field={activeColorModalField as Field} />
-                  )}
-                </Display>
-              </DraggableContent>
-              <ColorFooter />
-            </Container>
+            <Resizable
+              size={{ height, width }}
```

Comment:
suggestion for future (may ignore):  Looks like we're duplicating most of these props. We should probably create re-usable wrapper